src/main/java/core/basesyntax/DataReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReaderImpl implements DataReader {
+    @Override
+    public List<String> readFromFile(String path) {
+        List<String> lines = new ArrayList<>();
+
+        try (BufferedReader br = new BufferedReader(new FileReader(path));) {
+            String line = br.readLine();
+
+            while ((line = br.readLine()) != null) {

```

Comment:
The first line read by `br.readLine()` on line 15 is not added to the `lines` list. If the intention is to skip the first line, this is correct. Otherwise, consider adding it to the list before the while loop.